addons/mail/models/discuss/discuss_channel.py

```diff
@@ -487,8 +487,8 @@ def _notify_get_recipients(self, message, msg_vals, **kwargs):
         message_type = msg_vals.get('message_type', 'comment') if msg_vals else message.message_type
         pids = msg_vals.get('partner_ids', []) if msg_vals else message.partner_ids.ids
 
-        # notify only user input (comment or incoming / outgoing emails)
-        if message_type not in ('comment', 'email', 'email_outgoing'):
+        # notify only user input (comment, whatsapp messages or incoming / outgoing emails)
+        if message_type not in ('comment', 'email', 'email_outgoing', 'whatsapp_message'):
```

Comment:
Not the greatest way to have reference to `whatsapp` in `mail` module.
I know sometimes it makes the code easier to just break the modular pattern, but couldn't we simply have a method returning types allowed for notify to override in `whatsapp` module?

Reply:
Would say it is better to keep it short, not like we will have tons of overrides. Otherwise it is "yet another method people don't know it exists and won't use anyway" ^^ .